client/src/test/java/org/asynchttpclient/ClientStatsTest.java

```diff
@@ -0,0 +1,169 @@
+package org.asynchttpclient;
+
+import static org.asynchttpclient.Dsl.asyncHttpClient;
+import static org.asynchttpclient.Dsl.config;
+import static org.testng.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.testng.annotations.Test;
+
+/**
+ * Created by grenville on 9/25/16.
+ */
+public class ClientStatsTest extends AbstractBasicTest {
+
+    @Test(groups = "standalone")
+    public void testClientStatus() throws Throwable {
+        try (final DefaultAsyncHttpClient client = (DefaultAsyncHttpClient) asyncHttpClient(config().setKeepAlive(true).setPooledConnectionIdleTimeout(5000))) {
```

Comment:
no need to cast if we aim for AHC 2.1 and change the `AsyncHttpClient` interface.